docs: fix typo (#25127)
nuxt/nuxt: docs/3.api/3.utils/$fetch.md

:read-more{to="/docs/getting-started/data-fetching"}

> You can use `$fetch` for any methods that are executed only on client-side.

Reviewer: ```suggestion
You can use `$fetch` within any function that is executed only on client-side.
```
